tests/test_identifiers.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+
+import httpretty
+from factory import SubFactory
+from nose.tools import *  # noqa
+
+from tests.base import OsfTestCase
+from tests.factories import AuthUserFactory
+from tests.factories import ModularOdmFactory
+from tests.factories import RegistrationFactory
+from tests.test_addons import assert_urls_equal
+
+import furl
+from modularodm.storage.base import KeyExistsException
+
+from website import settings
+from website.identifiers.utils import to_anvl
+from website.identifiers.model import Identifier
+
+
+class IdentifierFactory(ModularOdmFactory):
+    FACTORY_FOR = Identifier
+
+    referent = SubFactory(RegistrationFactory)
+    category = 'carpid'
```

Comment:
This has to be unique, right? Maybe use `tests.factories.FakerAttribute` or a `Sequence`?